src/components/Rockets/Rockets.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import configureStore from 'redux-mock-store';
```

Comment:
- [ ] Good job testing your function for reliability :clap: but this test is failing due to a missing dependency in your project. Could you try running `npm i redux-mock-store` ? I think that resolves this issue.